includes/wc-attribute-functions.php

```diff
@@ -111,7 +111,7 @@ function wc_attribute_taxonomy_name_by_id( $attribute_id ) {
  * @return int
  */
 function wc_attribute_taxonomy_id_by_name( $name ) {
-	$name       = str_replace( 'pa_', '', $name );
+	$name       = str_replace( 'pa_', '', strtolower( $name ) );
```

Comment:
Should this use something like `sanitize_title` instead of `strtolower` in order to work with multiword attribute names (e.g. `wc_attribute_taxonomy_id_by_name( "Two Words" )`)?